docs/src/content/docs/guides/components.mdx

```diff
@@ -58,24 +58,32 @@ These components are available from the `@astrojs/starlight/components` package.
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 You can display a tabbed interface using the `<Tabs>` and `<TabItem>` components.
-Each `<TabItem>` must have a `label` to display to users.
+Each `<TabItem>` must have a `label` to display to users and can optionally include an `icon` attribute set to the name of [one of Starlight’s built-in icons](#all-icons).
```

Comment:
Tiny suggestion to break these into two sentences:

```suggestion
Each `<TabItem>` must have a `label` to display to users.
Use the optional `icon` attribute to include one of [Starlight’s built-in icons](#all-icons) next to the label.
```

Reply:
Notes are never a bother, always appreciated!